packages/style-engine/README.md

```diff
@@ -57,12 +57,19 @@ To enqueue a style for rendering in the site's frontend, the `$options` array re
 
 #### Usage
 
+As mentioned, `wp_style_engine_get_styles()` is useful whenever you wish to generate CSS and/or classnames from a **block's style object**. A good example is [using the Style Engine to generate block supports styles](https://github.com/WordPress/gutenberg/tree/HEAD/packages/style-engine/docs/using-the-style-engine-with-block-supports.md).
+
+In the following snippet, we're taking the style object from a block's attributes and passing it to the style engine to get the styles. By passing a `context` in the options, the Style Engine will store the styles for later retrieval, for example, should you wish to batch enqueue a set of CSS rules.
```

Comment:
The first "style engine" in this sentence is not in title case. Is it important to be consistent? If so, a bit further up in the description of `context` it's currently lowercase, too.

Reply:
Good call. Fixed.